spec/queries/my_facilities/registrations_query_spec.rb

```diff
@@ -72,20 +58,19 @@
           end
         end
 
-        specify { expect(registrations_query.registrations.count).to eq(included_timestamps.count) }
+        it 'counts registrations diagnosed with hypertension in the last_n months' do
```

Comment:
I think the `specify` is clean enough, no need to break it up and claim HTN explicitly